cybersixgill_actionable_alerts/assets/configuration/spec.yaml

```
@@ -0,0 +1,58 @@
+name: cybersixgill_actionable_alerts
+files:
+- name: cybersixgill_actionable_alerts.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+      - name: cl_id
+        required: true
+        description: The Client Id given by Cybersixgill
+        enabled:  true
+        value:
+          type: string
+          example:  clientid
+          display_default:  null
+      - name: cl_secret
+        required: true
+        description:  The Client Secret given by Cybersixgill
+        enabled:  true
+        value:
+          type: string
+          example:  client00000secret
+          display_default:  null
```

```suggestion
      - name: cl_secret
        required: true
        description:  The Client Secret given by Cybersixgill
        enabled:  true
        value:
          type: string
          display_default:  null
          secret: true
```
No need to display an example for a secret, and also should include `secret` to make sure the config value is  obscured.